tests/TestCase/View/Helper/FormHelperTest.php

```diff
@@ -2911,8 +2964,13 @@ public function testControlCustomization(): void
             'Field',
             '/label',
             'input' => [
-                'type' => 'text', 'name' => 'field',
-                'id' => 'field', 'class' => 'form-error',
+                'type' => 'text',
+                'name' => 'field',
+                'id' => 'field',
+                'class' => 'form-error',
+                'aria-invalid' => 'true',
+                // TODO this seems bad as the id doesn't exist.
+                'aria-describedby' => 'field-error',
```

Comment:
Maybe hardcode the `id` attribute in the custom `error` template? The `aria-describedby` option could receive a custom value too, after all that's something that should be tested anyways.